packages/mobile/src/identity/verification.ts

```diff
@@ -495,6 +497,13 @@ export function* requestAndRetrieveAttestations(
 
     // Check if we have a sufficient set now by fetching the new total set
     attestations = yield call(getActionableAttestations, attestationsWrapper, phoneHash, account)
+    if (features.SHORT_VERIFICATION_CODES) {
+      // we only support attestation service 1.1.0 and above for short codes
+      attestations = attestations.filter(
```

Comment:
What happens if one or two attestation services have an old version? We'd be missing attestations right? Should we ask for more in that case, is it unlikely to happen or is there somewhere else that's already handling that?

Reply:
Thanks for the question! If any of the services has older version we would ignore this attestation service and request more, cause outer while loop would not exit.